setup/gcp/installer/util/utils.py

```diff
@@ -298,3 +300,266 @@ def sanitize_conf_values(conf_values):
         if not conf_values[key]:
             conf_values[key] = '""'
     return conf_values
+
+
+def merge_object(base_obj, target_obj,
```

Comment:
I think that the arguments can be more descriptive:
```
def merge_object(merge_target, migration_target, ......)
```

Reply:
What about source_object and target_object? Merging target into source. I think merge_target and migration_target is bit hard to understand.